api/api.py

```diff
@@ -499,3 +501,30 @@ def bids(user, key):
             "Your bid will be confirmed once you scan the QR code.",
         ] + (["Started following the auction."] if started_following else []),
     })
+
+
+@api_blueprint.route('/api/stores/<string:twitter_username>/auctions', methods=['GET'])
+def store_auctions_view(twitter_username):
+    user = m.User.query.filter_by(twitter_username=twitter_username).first()
+    if user:
+        return jsonify({'auctions': [a.to_dict() for a in sorted(user.auctions.all(), key=lambda a: len(a.bids), reverse=True)]})
+
+
+@api_blueprint.route('/api/stores/<string:twitter_username>', methods=['GET'])
+def store_view(twitter_username):
+    if request.method == 'GET':
+        user = m.User.query.filter_by(twitter_username=twitter_username).first()
+        if not user:
+            return jsonify({}), 404
+        return jsonify({'user': user.to_dict()})
+
+
+@api_blueprint.route('/api/stores/<string:twitter_username>/store_info', methods=['PUT'])
```

Comment:
This seems useless, since the store info is part of the user.

Just use /api/users/me to save the necessary info.

Reply:
deleting this based on discussion in telegram